src/pages/community/2025/unitaryCON.mdx

```diff
@@ -4,50 +4,102 @@ title: 'unitaryCON'
 ---
 
 <p align="center">
-  <img src="/images/2025_unitaryCON.png" style="width: 50%" />
+  <img src="/images/2025_unitaryCON.png" style="width: 50%" alt="unitaryCON 2025 logo"/>
 </p>
 
-<div style='font-size: 1.5rem; line-height: 2rem;'>
-unitaryCON is an invitation-only collaborative workshop for the extended Unitary Foundation community. The workshop is an opportunity to share ongoing projects, connect with collaborators and supporters, and work with our community to advance the quantum open-source software ecosystem with the leading contributors from around the world.
-</div>
-<br/>
+<p class="text-3xl font-bold">unitaryCON - Open Source Quantum Technology Conference</p>
 
-<div>
+🗓️ September 2-4, 2025 | 📍 ABQ Convention Center 401 2nd St NW, Albuquerque, NM 87102
+
+unitaryCON is an invitation-only collaborative workshop for the extended Unitary Foundation community. The workshop is an opportunity to share ongoing projects, connect with collaborators and supporters, and work with our community to advance the quantum open-source software ecosystem with the leading contributors from around the world.
 
 Hosted at [IEEE Quantum Week 2025](https://qce.quantum.ieee.org/2025/), unitaryCON will provide a unique opportunity for Unitary Foundation advisors, grantees, staff, ambassadors and friends to gather, talk, learn, and share ideas about how to move the field forward.
 
 The workshop will feature talks and interactive activities, revolving around various topics in open source software for quantum computing and quantum technologies, including compilers, error mitigation and error correction toolkits, platforms for quantum computing, SDKs, cloud platforms, and open hardware in quantum tech.
 
 For information on the unitaryCON code of conduct, please visit our [GitHub](https://github.com/unitaryfoundation/unitary.foundation/blob/master/CODE_OF_CONDUCT.md).
 
-## unitaryCON event details
-- **Venue:** [**IEEE Quantum Week**](https://qce.quantum.ieee.org/2025/)
-Albuquerque, New Mexico, USA, at the Albuquerque Convention Center
-
-### **Schedule overview:**
-Coming Soon!
-
+<p class="text-2xl">unitaryCON event details</p>
+
+**Venue: [IEEE Quantum Week](https://qce.quantum.ieee.org/2025/)** Albuquerque, New Mexico, USA, at the Albuquerque Convention Center
+
+<p class="text-2xl">Schedule overview</p>
+
+<div class="overflow-x-auto">
```

Comment:
I've included this for review, but assume we might want to remove it before publishing/until we have the schedule set?